### ISSUE TITLE
detection score is low

### ISSUE THREAD

**Issue by LP940708** [mahyarnajibi/SNIPER#177]
I refer your code to my own dataset, and the bounding box is accurate, but the detection score is  low(I trained 7 epochs on my own dataset, but most of the detection score is lower than 0.5, rarely higher than 0.5), so I have to set a very low threshold.
Have you ever encountered this situation, can you give me some advice?
Thank you very much.

**Comment by szr712** [comment]
have you solved this problem?

**Comment by LP940708** [comment]
> have you solved this problem?

No, have you encountered this problem?

**Comment by szr712** [comment]
Yes, I have trained a model for my dataset. Same as you, the detection score is lower than 0.5.
However, the mean AP@0.5 is accurate.
